frontend/test/ui/conftest.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from foxpuppet import FoxPuppet
+
+
+@pytest.fixture
+def foxpuppet(selenium):
+    return FoxPuppet(selenium)
+
+
+@pytest.fixture
+def capabilities(capabilities):
+    capabilities['marionette'] = True
+    capabilities['acceptInsecureCerts'] = True
+    return capabilities
+
+
+@pytest.fixture
+def firefox_profile(firefox_profile):
+    firefox_profile.set_preference(
+        'extensions.install.requireBuiltInCerts', 'false')
+    firefox_profile.set_preference('xpinstall.signatures.required', 'false')
+    firefox_profile.set_preference('extensions.webapi.testing', 'true')
+    firefox_profile.set_preference('testpilot.env', 'local')
+    return firefox_profile
```

Comment:
Rather than passing an entire profile, you can now pass individual preferences. For example:

```python
@pytest.fixture
def firefox_options(firefox_options):
    firefox_options.set_preference('extensions.install.requireBuiltInCerts', False)
    firefox_options.set_preference('xpinstall.signatures.required', False)
    firefox_options.set_preference('extensions.webapi.testing', True)
    firefox_options.set_preference('testpilot.env', 'local')
    return firefox_options
```

The current pytest-selenium will still send a profile, but a future version will only do so when a profile object is used. By using `firefox_options` now, this will reduce the new session payload in the future.